Automattic/woocommerce-payments
# refactor: add survey modal cancel buttons

Fixes https://github.com/Automattic/woocommerce-payments/issues/2546

#### Changes proposed in this Pull Request









Before:
![Screen Shot 2021-07-23 at 3 16 38 PM](https://user-images.githubusercontent.com/273592/126836919-38a6c084-ccc1-4602-a756-9af2fa423f2c.png)
![Screen Shot 2021-07-23 at 3 16 33 PM](https://user-images.githubusercontent.com/273592/126836920-83333f7d-aa7c-403e-8297-03d883ca8f70.png)


After:
![Screen Shot 2021-07-23 at 3 15 44 PM](https://user-images.githubusercontent.com/273592/126836944-5259ad05-f42d-46ab-9ce9-08481a8c57b5.png)
![Screen Shot 2021-07-23 at 3 15 50 PM](https://user-images.githubusercontent.com/273592/126836943-62f263cf-bc88-4c31-b454-0bd91844c000.png)


Adding "cancel" buttons to the survey modals.

#### Testing instructions





- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Enable UPE
- Disable or send feedback
- Modals should now have a "cancel" action

-------------------
